charts/gitprime-app/v4.1.0/values.yaml

```diff
@@ -0,0 +1,408 @@
+        value: local_tasks
+    resources:
+      limits:
+        cpu: "4"
+        memory: 8Gi
+      requests:
+        cpu: "2"
+        memory: 4Gi
+
+  prWorker:
+    replicaCount: 1
+    javaOptions: "-d64 -Xms2g -Xmx4g"
+    prThreadsPerListener: 1
+    prListeners: 1
+    logLevel: INFO
+    environment:
+      - name: GP_REPROCESS_LISTENERS
+        value: 0
+      - name: GP_REPO_DELETE_LISTENERS
+        value: 0
+      - name: GP_AOD_LISTENERS
+        value: 0
+      - name: GP_NEW_LISTENERS
+        value: 0
+      - name: GP_INCREMENTAL_LISTENERS
+        value: 0
+      - name: GP_TICKET_LISTENERS
+        value: 0
+      - name: GP_TICKET_WEBHOOK_LISTENERS
+        value: 0
+      - name: GP_PR_WEBHOOK_LISTENERS
+        value: 1
+      - name: GP_PR_WEBHOOK_WORKERS_PER_LISTENER
+        value: 1
+      - name: GP_BP_LISTENERS
+        value: 0
+      - name: GP_MAX_REPO_PROCESSING_SECONDS
+        value: 2700
+      - name: GP_MAX_CONCURRENT_PROJECTS_PER_ORG
+        value: 40
+      - name: GP_SCHEDULED_TASK_SET
+        value: local_tasks
+    resources:
+      limits:
+        cpu: "4"
+        memory: 8Gi
+      requests:
+        cpu: "2"
+        memory: 4Gi
+
+  reprocessWorker:
+    replicaCount: 1
+    javaOptions: "-d64 -Xms2g -Xmx4g"
+    reprocessThreadsPerListener: 1
+    reprocessListeners: 1
+    logLevel: INFO
+    environment:
+      - name: GP_REPROCESS_LISTENERS
+        value: 1
+      - name: GP_REPO_DELETE_LISTENERS
+        value: 0
+      - name: GP_AOD_LISTENERS
+        value: 0
+      - name: GP_NEW_LISTENERS
+        value: 0
+      - name: GP_INCREMENTAL_LISTENERS
+        value: 0
+      - name: GP_TICKET_LISTENERS
+        value: 0
+      - name: GP_TICKET_WEBHOOK_LISTENERS
+        value: 0
+      - name: GP_PR_LISTENERS
+        value: 0
+      - name: GP_PR_WEBHOOK_LISTENERS
+        value: 0
+      - name: GP_BP_LISTENERS
+        value: 0
+      - name: GP_MAX_REPO_PROCESSING_SECONDS
+        value: 2700
+      - name: GP_SCHEDULED_TASK_SET
+        value: local_tasks
+    resources:
+      limits:
+        cpu: "4"
+        memory: 8Gi
+      requests:
+        cpu: "2"
+        memory: 4Gi
+
+  bpWorker:
+    replicaCount: 1
+    javaOptions: "-d64 -Xms2g -Xmx4g"
+    bpThreadsPerListener: 1
+    bpListeners: 1
+    logLevel: INFO
+    environment:
+      - name: GP_REPROCESS_LISTENERS
+        value: 0
+      - name: GP_REPO_DELETE_LISTENERS
+        value: 0
+      - name: GP_AOD_LISTENERS
+        value: 0
+      - name: GP_NEW_LISTENERS
+        value: 0
+      - name: GP_INCREMENTAL_LISTENERS
+        value: 0
+      - name: GP_TICKET_LISTENERS
+        value: 0
+      - name: GP_TICKET_WEBHOOK_LISTENERS
+        value: 0
+      - name: GP_PR_WEBHOOK_LISTENERS
+        value: 0
+      - name: GP_PR_WEBHOOK_WORKERS_PER_LISTENER
+        value: 0
+      - name: GP_BP_LISTENERS
+        value: 1
+      - name: GP_BP_WEBHOOK_WORKERS_PER_LISTENER
+        value: 1
```

Comment:
I don't think we need any webhook listeners for Background processing

Reply:
Good catch.  I knew there would be at least one copy/paste error...

Reply:
Fixed.